src/test-data/cycle.c

```diff
@@ -0,0 +1,66 @@
+#include "blake2b.h"
+#include "blockchain.h"
+#include "ckb_syscalls.h"
+#include "secp256k1_helper.h"
+#include "secp256k1_lock.h"
+#define SCRIPT_SIZE 32768
+
+int read_args(uint64_t *loop)
+{
+    int ret;
+    uint64_t len = 0;
+//    char buf[4096];
+
+    /* Load args */
+    unsigned char script[SCRIPT_SIZE];
+    len = SCRIPT_SIZE;
+    ret = ckb_load_script(script, &len, 0);
```

Comment:
I think load from witness would be more easier, so that `blockchain.h` also not needed.